src/MyMind.Web/Areas/Emotions/Controllers/EmotionsController.cs

```diff
@@ -0,0 +1,32 @@
+﻿using System.Linq;
+using Microsoft.AspNetCore.Mvc;
+using MongoDB.Driver;
+using MyMind.Web.Database;
+using MyMind.Web.Domain.Emotions;
+
+namespace MyMind.Web.Areas.Emotions
+{
+    public class EmotionsController : BaseApiController
+    {
+        private readonly ISessionFactory _sessionFactory;
+
+        public EmotionsController(ISessionFactory sessionFactory)
+            => this._sessionFactory = sessionFactory;
```

Comment:
nawet jak kod mieści się w jednej linii, to zazwyczaj lepiej wrzucać go w klamry, tak jest czytelniej

Reply:
ok